### PR TITLE
2 New Commands 

### PR DESCRIPTION
**Pull request by pgamerxdev**
I just added 2 more commands - pause and resume in example.
I will be adding more over time

### CODE REVIEW

File: commands/pause.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    name: "pause",
+    aliases: ["pause", "hold"],
+    run: async (client, message, args) => {
+        if (!message.member.voice.channel) return message.channel.send(`${client.emotes.error} | You must be in a voice channel!`)
+        let queuethere = client.distube.getQueue(message);
+         if (!queuethere) return message.channel.send(`${client.emotes.error} There is nothing in the queue right now!`)
+         client.distube.pause(message)
```
Comment (skick1234) [inline commands/pause.js:8]: I think we should unpause the song if it have been paused already.
```suggestion
        let queue = client.distube.getQueue(message)
        if (!queue) return message.channel.send(`${client.emotes.error} | There is nothing in the queue right now!`)
        if (queue.pause) {
            client.distube.resume(message)
            return message.channel.send("Resumed the song for you :)")
        }
        client.distube.pause(message)
```

File: commands/resume.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    name: "resume",
+    aliases: ["resume", "unpause"],
+    run: async (client, message, args) => {
+        if (!message.member.voice.channel) return message.channel.send(`${client.emotes.error} | You must be in a voice channel!`)
+        let queuethere = client.distube.getQueue(message);
+         if (!queuethere) return message.channel.send(`${client.emotes.error} There is nothing in the queue right now!`)
+         client.distube.resume(message)
```
Comment (skick1234) [inline commands/resume.js:8]: Linting
```suggestion
        let queue = client.distube.getQueue(message)
        if (!queue) return message.channel.send(`${client.emotes.error} | There is nothing in the queue right now!`)
        client.distube.resume(message)
```

**Review by skick1234 — CHANGES_REQUESTED** [review]

**Comment by pgamerxdev** [comment]
Everything looks fine no 
Kindly recheck

**Review by skick1234 — APPROVED** [review]
Thanks for the pr 😉

**Comment by pgamerxdev** [comment]
No problem buddy ;)

**Pull request merged**